starfish/experiment/__init__.py

```diff
@@ -64,6 +64,15 @@ def __init__(
         else:
             self._auxiliary_images = dict()
 
+    def __repr__(self):
+        auxiliary_images = '\n    '.join(f'{k}: {v}' for k, v in self._auxiliary_images.items())
+        return (
+                f"<starfish.FieldOfView>\n"
```

Comment:
FOVs have names.  why not use it?

Reply:
When I tried it, I got: 
```python
>>> fov.name is None
True
```

Reply:
Did you have a single FOV experiment?  They are not required to have names.

Reply:
Yea, I did. My thoughts are "eliminate this inconsistency, everything should have names".